src/main/scripts/startMZmine_Linux.sh

```diff
@@ -1,48 +1,70 @@
 #!/bin/sh
 
-# The HEAP_SIZE variable defines the Java heap size in MB.
-# That is the total amount of memory available to MZmine 2.
-# By default we set this to half of the total memory or 
-# 2048 MB less than the total memory.
-# Feel free to adjust the HEAP_SIZE according to your needs.
+# *****************************************
+# Optional values - Please modify as needed
+# *****************************************
 
+# Total amount of memory in MB available to MZmine 2.
+# AUTO = automatically determined
+# Default: AUTO
+HEAP_SIZE=AUTO
+
+# Location where temporary files will be stored.
+# Default: /tmp
+TMP_FILE_DIRECTORY=/tmp
+
+# Allow MZmine to send anonymous statistics on the module usage? 
+# 1 = activated, 0 = deactivated
+# Default: 1
+export MZMINE_STATISTICS=1
+
+# It is usually not necessary to modify the JAVA_COMMAND parameter, but if you like to run
+# a specific Java Virtual Machine, you may set the path to the java command of that JVM
+JAVA_COMMAND=java
+
+# ********************************************
+# You don't need to modify anything below here
+# ********************************************
+
+
+
+
+# ***********************************
+# Auto detection of accessible memory
+# ***********************************
+
+# By default we set the HEAP_SIZE to 1024 MB on 32-bit systems. On 64-bit systems we 
+# either set it to half of the total memory or 2048 MB less than the total memory.
 echo "Checking physical memory size..."
 TOTAL_MEMORY=`free -b | awk '/Mem:/ { print int($2 / 1024^2) }'`
 echo "Found $TOTAL_MEMORY MB memory"
 
-if [ "$TOTAL_MEMORY" -gt 4096 ]; then
+if [ "$HEAP_SIZE" == "AUTO" ]; then
```

Comment:
Hi Thomas,
I think it should be a single equal sign (=), not ==.
As I posted in the previous message:

     s1 = s2       True if the strings s1 and s2 are identical.